From 87f67d504bba8274f96cc77705fc8e37e467d0ed Mon Sep 17 00:00:00 2001
From: Yuya Hamamachi <yuya.hamamachi.sx@renesas.com>
Date: Thu, 4 Aug 2022 17:50:04 +0900
Subject: [PATCH 2/3] starterkit/config/mot: Change to use 64bit mode

Signed-off-by: Yuya Hamamachi <yuya.hamamachi.sx@renesas.com>
---
 starterkit/config/mot | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/starterkit/config/mot b/starterkit/config/mot
index c8f9980..ec88a7a 100644
--- a/starterkit/config/mot
+++ b/starterkit/config/mot
@@ -5,9 +5,9 @@
 ; We use 32bit CPLD mode for all Starterkit boards.
 ;
 dir_mot	= "starterkit"
-mot	= "AArch32_Flash_writer_SCIF_DUMMY_CERT_E6300400_ULCB.mot"
+mot	= "AArch64_Flash_writer_SCIF_DUMMY_CERT_E6300400_ULCB.mot"
 
 num_cpld_cmd = 2
 strdim cpld_cmd num_cpld_cmd
-cpld_cmd[0] = "cpld write 0x00 0x802181fe"
+cpld_cmd[0] = "cpld write 0x00 0x8021813e"
 cpld_cmd[1] = "cpld write 0x80 0x01"
-- 
2.34.1

